doc/make.bat

```diff
@@ -1,263 +1,263 @@
-@ECHO OFF

```

Comment:
Ah, you'll have to drop this change (from https://github.com/ansible/molecule/pull/2004).